code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -576,8 +573,9 @@
 		/datum/reagent/consumable/clownstears,
 		/datum/reagent/toxin/minttoxin,
 		/datum/reagent/consumable/ethanol/atomicbomb,
-		/datum/reagent/drug/aphrodisiac,
-		/datum/reagent/drug/aphrodisiacplus
+		/datum/reagent/consumable/ethanol/thirteenloko,
+		/datum/reagent/consumable/ethanol/changelingsting,
+		/datum/reagent/consumable/ethanol/whiskey_cola
```

Comment:
This one is not even that valuable, it's litterally just made of whiskey and cola. Could you please remove it?

Reply:
done